python-interface/src/MiniBotScripts/Buddybot.py

```diff
@@ -0,0 +1,190 @@
+import Note_Library as nl
+import live_audio_sample as las
+import random
+
+#Used for individual emotion testing 
+TEST_EMOTION = 0; #TODO remove once emotions finalized
+
+#Noise frequency thresholds #TODO check if correct
+SHOUT_THRESHOLD = 850
+SPEAK_THRESHOLD = 450
+
+# Should change at most by factor of 1 every second
+HAPPY_FACTOR = 10
+SAD_FACTOR = 200
+SURPRISE_FACTOR = 1
+ANGER_FACTOR = 50 #Currently not implemented
+
+SAMPLE_FILE = "sample.wav"
+
+#Global emotion counters
+bot_happy = 0
+bot_sad = 0
+bot_surprised = 0
+bot_angry = 0
+
+def randomEmotion():
+    return random.randint(0,3)
+
+def neutralNoise():
+    nl.playNote(nl.G5,0.15)
+
+def neutralNoise2():
+    nl.playNote(nl.G4,0.15)
+
+def happySong():
+    print("First note")        
+    nl.playNote(nl.G6,0.15)
+    print("Second note")
+    nl.playNote(nl.A6,0.15)
+    print("Third note")
+    nl.playNote(nl.G6,0.15)
+    nl.playNote(nl.A6,0.15)
+    nl.playNote(nl.B6,0.25)
+
+def showHappy(bot):
+    happySong()
+    bot.move_forward(25)
+    bot.wait(3)
+    bot.move_backward(25)
+    bot.wait(3)
+    bot.turn_clockwise(25)
+    bot.wait(5)
+    bot.turn_counter_clockwise(25)
+    bot.wait(5)    
+
+def sadSong():
+    nl.playNote(nl.G6,0.25)
+    nl.playNote(nl.D6_SHARP,0.25)
+    nl.playNote(nl.G3,0.5)
+    nl.playNote(nl.G2,1)
+
+def showSad(bot):
+    sadSong()
+    bot.move_backward(10)
+    bot.wait(5)
+
+def surpriseSong():
+    nl.playNote(nl.B6,0.10)
+    nl.playNote(nl.B6,0.10)
+    nl.playNote(nl.B6,0.10)
+    nl.playNote(nl.B6,0.10)
+
+def showSurprise(bot):
+    surpriseSong()
+    bot.move_backward(80)
+    bot.wait(0.75)
+    bot.move_backward(80)
+    bot.wait(0.75)
+    bot.move_backward(80)
+    bot.wait(0.75)
+
+def angerSong():
+    nl.playNote(nl.G3,0.10)
```

Comment:
also, you don't need to change it since this is mostly an example file, but you could have saved a lot of code by defining an array of these notes and looping.